ui/components/app/account-menu/account-menu.test.js

```diff
@@ -96,7 +96,7 @@ describe('Account Menu', () => {
 
     it('render imported account label', () => {
       const importedAccount = wrapper.find('.keyring-label.allcaps');
-      expect(importedAccount.text()).toStrictEqual('imported');
+      expect(importedAccount.text()).toStrictEqual('[imported]');
```

Comment:
We discussed this with @danjm and due to the way we wrap our `useI18nContext` around the provider, the square brackets are needed in this case